src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1,49 +1,49 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "Static Analysis Results Format (SARIF) Version 2.0.0-csd.2.beta.2019-01-24 JSON Schema",
-  "description": "Static Analysis Results Format (SARIF) Version 2.0.0-csd.2.beta-2019-01-24 JSON Schema: a standard format for the output of static analysis tools.",
-  "additionalProperties": false,
-  "type": "object",
-  "properties": {
-
-    "$schema": {
-      "description": "The URI of the JSON schema corresponding to the version.",
-      "type": "string",
-      "format": "uri"
-    },
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Static Analysis Results Format (SARIF) Version 2.0.0-csd.2.beta.2019-01-24 JSON Schema",
+    "description": "Static Analysis Results Format (SARIF) Version 2.0.0-csd.2.beta-2019-01-24 JSON Schema: a standard format for the output of static analysis tools.",
+    "additionalProperties": false,
+    "type": "object",
+    "properties": {
 
-    "version": {
-      "description": "The SARIF format version of this log file.",
-      "enum": [ "2.0.0-csd.2.beta.2019-02-20" ]
-    },
+        "$schema": {
+            "description": "The URI of the JSON schema corresponding to the version.",
+            "type": "string",
```

Comment:
Take the external properties schema and create an actual schema file for this scenario. Name it "sarif-external-property-file.json".